.github/workflows/verify.yml

```diff
@@ -5,7 +5,7 @@ on:
   pull_request:
 
 env:
-  NODE_VERSION: "14"
+  NODE_VERSION: "16.10.0"
```

Comment:
https://github.com/Qiskit/qiskit.org/pull/2919 makes now possible to set all 16 versions without the need to specify 16.10.

Reply:
Resolved w/ https://github.com/Qiskit/qiskit.org/pull/2912/commits/c085b6c1fd6799610247849abee843554c6fe9a8